dohproxy/proxy.py

```diff
@@ -136,12 +140,14 @@ def stream_complete(self, stream_id: int):
             self.return_400(stream_id, body=e.body())
             return
 
+        clientip = self.transport.get_extra_info('peername')[0]
         self.logger.info(
-            '[HTTPS] Received: {} Peer {}'.format(
-                utils.dnsmsg2log(dnsq),
-                self.transport.get_extra_info('peername'),
+            '{} {}'.format(
```

Comment:
We still need to keep the `[HTTPS]` or `[DNS]` and wether it is received or sent at the moment. This helps in understanding where/when the logs are being written. This applies for each log entries you have edited.